src/styles/reset.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't use * for reset something, list all elemts


```
ul, li, p { mar: 0 }
```

Reply:
ok